### ISSUE TITLE
Enchantment table summons lightning

### ISSUE THREAD

**Issue by MyOnionsHaveLayers** [ITsMrToad/PalladiumMod#15]
Not sure if it's toad lib or palladium but on the currently most recent 1,20,1 forge version, by interacting with an enchantment table, a lightning bolt will be summoned ontop and hurt you. i don't know if this is an easter egg but i can't enchant anymore so i have removed the mod.

**Comment by ITsMrToad** [comment]
No, it's an easter egg. I testing the features and ToadLib and and forgot to remove this, I will post 1.1.5 now, I apologize for the inconvenience this has caused.
